components/Templates/class-pods_templates.php

```diff
@@ -221,6 +221,15 @@ public function add_metaboxes( $slug, $post = false ) {
 				'groups' => array(),
 			)
 		);
+		add_meta_box(
+			'pod_reference', __( 'Magic Tag Reference', 'pods' ), array(
```

Comment:
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: 1c0d3fc2ebfa87e21348bfb2133f0c3f</sub>